ai_chatbots/tools.py

```diff
@@ -186,6 +186,20 @@ class SearchContentFilesToolSchema(pydantic.BaseModel):
     )
 
 
+class VideoGPTToolSchema(pydantic.BaseModel):
+    """Schema for searching MIT contentfiles related to a particular course."""
```

Comment:
"""Schema for searching MIT contentfiles related to a particular video transcript.""" ?